### ISSUE TITLE
enableDragAndDrop doesn't seem to be used

### ISSUE THREAD

**Issue by thomas-shares** [adazzle/react-data-grid#600]
**WHICH VERSION OF REACT ARE YOU USING?**
```
Officially Supported:
[X] v0.14.x
```
```
Community Supported:
[ ] v15.0.x
```
**WHICH BROWSER ARE YOU USING?**
```
Officially Supported:
[ ] IE 9 / IE 10 / IE 11
[ ] Edge
[ ] Chrome
```
```
Should work:
[X] Firefox
[ ] Safari
```
--------
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report
[X] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/adazzle/react-data-grid/blob/master/CONTRIBUTING.md
```
**Current behavior**
We are currently experiencing issue #225 and as a work around we were thinking about disabling dragging completely. A investigation of the docs revealed the enableDragAndDrop option and setting this to 'false' didn't give us the expected behaviour. We did a grep over the code and it looks like that the enableDragAndDrop flag isn't used anywhere in the code and thus not functional.

**Expected/desired behavior**
If react-data-grid is initialised with enableDragAndDrop set to false one would expect that dragging would not be possible. And this in turn would work as a work around of issue #225.

**Reproduction of the problem**
NA

**What is the expected behavior?**
NA

**What is the motivation / use case for changing the behavior?**
Initial motivation for us would be to workaround issue #225, but also to make the code match the expectation as set in the documentation.

**Comment by thomas-shares** [comment]
ps. I am willing to help with this, but don't have enough JS experience for this at the moment.

**Comment by thomas-shares** [comment]
I have added a PR that allows the user to turn off DraggingAndDropping. Unfortunately this doesn't address issue #225 .

**Comment by thomas-shares** [comment]
@diogofcunha  as issue #225 has now been fixed (issue #603) do we still want to pursue this issue and its associated PR? 

Please advice.

Thomas
